scripts/d.rast.edit/d.rast.edit.py

```diff
@@ -666,7 +666,7 @@ def get_color(self, val):
             if val not in self.colors:
                 try:
                     self.force_color(val)
-                except:
+                except ValueError:
```

Comment:
That's not going to work. Looking at the entire method, wx.Image doesn't have get method, so this is not functioning code. Please remove the entire force_color method and adjust accordingly.